model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city.name
+        stability: experimental
+        type: string
+        brief: >
+          City name.
+        examples: [ 'Montreal', 'Berlin' ]
+      - id: continent.code
+        stability: experimental
+        brief: >
+          Two-letter code representing continent’s name.
+        type:
+          members:
+            - id: 'af'
+              stability: experimental
+              value: 'AF'
+              brief: 'Africa'
+            - id: 'an'
+              stability: experimental
+              value: 'AN'
+              brief: 'Antarctica'
+            - id: 'as'
+              stability: experimental
+              value: 'AS'
+              brief: 'Asia'
+            - id: 'eu'
+              stability: experimental
+              value: 'EU'
+              brief: 'Europe'
+            - id: 'na'
+              stability: experimental
+              value: 'NA'
+              brief: 'North America'
+            - id: 'oc'
+              stability: experimental
+              value: 'OC'
+              brief: 'Oceania'
+            - id: 'sa'
+              stability: experimental
+              value: 'SA'
+              brief: 'South America'
+      - id: continent.name
+        stability: experimental
+        type: string
+        brief: >
+          Name of the continent.
+        examples: [ 'North America', 'Europe' ]
+      - id: country.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://en.wikipedia.org/wiki/ISO_3166-1#Codes)).
+        examples: [ 'CA' ]
+      - id: country.name
+        stability: experimental
+        type: string
+        brief: >
+          Country name.
+        examples: [ 'Canada' ]
+      - id: location.lon
+        stability: experimental
+        type: double
+        brief: >
+          Longitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ -73.614830 ]
+      - id: location.lat
+        stability: experimental
+        type: double
+        brief: >
+          Latitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ 45.505918 ]
+      - id: name
```

Comment:
Since this is based on ECS, I'd prefer to not break compatibility on this field because of this. So I'd keep it as `name` for backwards compatibility with ECS.

Reply:
would this attribute be used by the collector processor? If not and we don't have a use-case for it in otel yet, could we postpone adding it?

Reply:
This one I don't see being used - @rogercoll do you know if this field is used currently? If not, I'm ok to remove this.

Reply:
This field is not used in the collector processor atm, neither in the Elastic's geoip processor: https://www.elastic.co/guide/en/elasticsearch/reference/current/geoip-processor.html

I agree to postpone the addition until we find a use case :+1:

Reply:
Thanks - removed `name`.